2024-10-29 20:34:12,062 - instabot version: 0.117.0 (bot) - INFO - Instabot version: 0.117.0 Started
2024-10-29 20:34:12,062 - instabot version: 0.117.0 (bot) - DEBUG - Bot imported from C:\Users\hp\AppData\Local\Programs\Python\Python312\Lib\site-packages\instabot\bot\bot.py
2024-10-29 20:34:12,062 - instabot version: 0.117.0 (api_login) - INFO - Not yet logged in starting: PRE-LOGIN FLOW!
2024-10-29 20:34:12,515 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/get_prefill_candidates/ returned response: <Response [200]>
2024-10-29 20:34:13,013 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: qe/sync/ returned response: <Response [200]>
2024-10-29 20:34:13,598 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: launcher/sync/ returned response: <Response [200]>
2024-10-29 20:34:14,099 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/contact_point_prefill/ returned response: <Response [200]>
2024-10-29 20:34:16,399 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/login/ returned response: <Response [400]>
2024-10-29 20:34:16,399 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"challenge_required","challenge":{"url":"https://i.instagram.com/challenge/AXEfnLd7C-ZevDN1J6vzjCmUQp5ouJIi-XnS_xpTNvIP61ZEAotBZmFf-Hw5ptetKBwwQac/AfwFkvXAB0ARTrxrDt7NOYooGiRe2HQ5o8whALLMUPKd9npCrYbpttJJeZXLELclT-4rIjLaIYPn_Q/","api_path":"/challenge/AXEfnLd7C-ZevDN1J6vzjCmUQp5ouJIi-XnS_xpTNvIP61ZEAotBZmFf-Hw5ptetKBwwQac/AfwFkvXAB0ARTrxrDt7NOYooGiRe2HQ5o8whALLMUPKd9npCrYbpttJJeZXLELclT-4rIjLaIYPn_Q/","hide_webview_header":true,"lock":true,"logout":false,"native_flow":true,"flow_render_type":0},"status":"fail","error_type":"checkpoint_challenge_required"}'
2024-10-29 20:34:16,399 - instabot version: 0.117.0 (api) - ERROR - Request returns 400 error!
2024-10-29 20:34:16,399 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: challenge_required
2024-10-29 20:34:16,399 - instabot version: 0.117.0 (api) - INFO - Error type: checkpoint_challenge_required
2024-10-29 20:34:20,575 - instabot version: 0.117.0 (api_photo) - ERROR - Photo Upload failed with the following response: <Response [403]>
2024-10-29 20:34:20,575 - instabot version: 0.117.0 (bot_photo) - INFO - Photo 'images/motivation_5404.png' is not uploaded.
2024-10-29 20:34:20,575 - instabot version: 0.117.0 (bot) - INFO - Bot stopped. Worked: 0:00:08.517737
2024-10-29 20:34:20,575 - instabot version: 0.117.0 (bot_checkpoint) - DEBUG - Saving Checkpoint file to: E:\Projects\Instagram Motivational Bot\instagram_motivation_bot/config/top1m_indset.checkpoint
2024-10-29 20:34:20,575 - instabot version: 0.117.0 (bot) - INFO - Total requests: 5
